augur/tasks/init/celery_app.py

```diff
@@ -216,6 +215,10 @@ def setup_periodic_tasks(sender, **kwargs):
         logger.info(f"Scheduling update of collection weights on midnight on even numbered days.")
         sender.add_periodic_task(crontab(0, 0,day_of_month='2-30/2'),augur_collection_update_weights.s())
 
+        cloning_interval = 5*60
```

Comment:
I think adding the cloning as a periodic task will cause problems with cloning multiple things at once and with having multiple instances of this task active at once. The solution to this, in my opinion, is to make it so that the task schedules itself at the end of its loop when it runs out of repos to clone on a delay. That way, we would never run more than once instance of the task at once but we would still check for repos to clone every five minutes.